test/p2p/sanity.spec.ts

```diff
@@ -41,17 +44,22 @@ describe('P2P Sanity Tests', () => {
     nodeTwo = new Xud();
 
     await Promise.all([nodeOne.start(nodeOneConfig), nodeTwo.start(nodeTwoConfig)]);
+
+    await nodeOne['db'].models.Node.truncate();
```

Comment:
I put in this same fax in #357 - which maybe I should just merge at this point because it's been open for a while and you've approved it - but does this actually work? This is after nodeOne has been initialized, meaning it should have loaded the nodes from Node table at this point. I guess this is moot given the other PR but I thought I'd ask.